partcad/src/partcad/lint/python.py

```
@@ -1,12 +1,72 @@
+                    raise
+                raise LintDependencyError(
+                    "Python linting requires an optional dependency ('ruff') that is not installed. "
+                    + RUFF_INSTALL_HINT
+                ) from e
+
+        if ruff_bin is None:
+            try:
+                ruff_bin = ruff_main.find_ruff_bin()
+            except FileNotFoundError as e:
+                raise LintDependencyError(
+                    f"The 'ruff' package is installed but its executable was not found ({str(e)}). "
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)